sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/LogsHelperTests.cs

```diff
@@ -87,13 +87,11 @@ public void MessageIsSetToFormattedMessageWhenIncludeFormattedMessageIsSet()
             var message = LogsHelper.GetMessageAndSetProperties(logRecords[0], properties);
 
             Assert.Equal("Hello from tomato 2.99.", message);
-            Assert.True(properties.TryGetValue("OriginalFormat", out string value));
```

Comment:
nit: consider adding this line: `Assert.False(properties.ContainsKey("OriginalFormat"));`

Reply:
This could lead to confusion because we view the presence of 'OriginalFormat' as a bug.